src/app/prevalence/components/PrevalenceSideContent.tsx

```diff
@@ -1,5 +1,233 @@
 import React from "react";
+import {
+    Box,
+    Button,
+    Checkbox,
+    FormControl,
+    FormControlLabel,
+    FormGroup,
+    FormLabel,
+    Accordion,
+    AccordionSummary,
+    AccordionDetails,
+    Typography,
+    styled,
+} from "@mui/material";
+import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
+import FilterListIcon from "@mui/icons-material/FilterList";
+import { useSelection } from "./SelectionContext";
+import { useTranslation } from "react-i18next";
+
+const StyledAccordion = styled(Accordion)(({ theme }) => ({
+    marginBottom: theme.spacing(4),
+    backgroundColor: "white",
+    borderBottom: `1px solid ${theme.palette.divider}`,
+    borderRight: `1px solid ${theme.palette.divider}`,
+    boxShadow: "none",
+    "&:before": {
+        display: "none",
+    },
+    "&.Mui-expanded": {
+        margin: 0,
+    },
+}));
+
+const microorganismOptions = [
+    "BAYLISASCARIS PROCYONIS",
+    "CLOSTRIDIOIDES DIFFICILE",
+    "DUNKER SCHER MUSKELEGEL",
+    "HEPATITIS A VIRUS",
+    "HEPATITIS E VIRUS",
+    "NOROVIRUS",
+    "PRÄSUMTIVER BACILLUS CEREUS",
+    "VIBRIO SPP",
+    "YERSINIA ENTEROCOLITICA",
+    "CAMPYLOBACTER_SPP",
+    "ESBL_AMPC_E_COLI",
+    "LISTERIS_MONOCYTOGENES",
+    "MRSA",
+    "SALMONELLA_SPP",
+    "STEC",
+    "CARBA_E_COLI",
+    "ENTEROCOCCUS_SPP",
+];
+
+const animalSpeciesOptions = [
+    "GEFLUEGEL",
+    "HUHN",
+    "KLEINEWIEDERKAEUER",
+    "MEERESFRUECHTE",
+    "NUTZTIERE",
+    "PFLANZLICHELEBENSMITTEL",
+    "PUTE",
+    "REHWILD",
+    "RIND",
+    "SCHWEIN",
+    "SCHWEINUNDRIND",
+    "SUESSWASSERFISCHE",
+    "WIEDERKAEUER",
+    "WILDGEFLUEGEL",
+    "WILDSCHWEIN",
+    "WILDWIEDERKAEUER",
+];
 
 export function PrevalenceSideContent(): JSX.Element {
-    return <></>;
+    // This already has a return type specified.
+    const {
```

Comment:
Please encapsulate all the functionality into a hook.  The PrevalenceSideContent component should then use the hook to get model values and actions (functions) which are then inserted into the jsx.